discord-impl/src/main/java/net/starype/quiz/discordimpl/command/RoundAddCommand.java

```diff
@@ -69,23 +70,35 @@ public void execute(CommandContext context) {
 
         conditions.put(
                 () -> args.length < 2,
-                "You must specify the type of round you wish to queue");
+                "You must specify the type of round you wish to queue (either race, classical or individual)");
 
         conditions.put(
-                () -> args.length >= 3 && (args[2].length() != 1 || !Character.isDigit(args[2].charAt(0))),
-                "Second argument must be a number between 0 and 9");
+                () -> args.length >= 3 && asRoundCount(args[2]).isEmpty(),
+                "Second argument must be a number between 1 and 20");
 
         return conditions;
     }
 
+    private static Optional<Integer> asRoundCount(String arg) {
+        try {
+            int value = Integer.parseInt(arg);
+            if(value > 0 && value < 21) {

```

Comment:
We could introduce dedicated constant for the maximum amount of round that could be created

Reply:
Yes indeed